test/functional/test_framework/mempool_util.py

```diff
@@ -92,6 +93,14 @@ def send_batch(fee):
         send_batch(fee)
     tx_sync_fun() if tx_sync_fun else test_framework.sync_mempools()  # sync after all evictions
 
+    # if mempool is almost full (<10k usage bytes left), submit one extra middle-sized tx,
+    # in order to evict a large filler tx and leave some room; this will enable tests to submit
+    # txs with just the mempoolminfee without immediate eviction ("mempool full" error)
+    if node.getmempoolinfo()['usage'] >= 4_990_000:
+        ephemeral_miniwallet.send_self_transfer(from_node=node, fee=num_of_batches * (base_fee / 2),
+                                                utxo_to_spend=confirmed_utxos.pop(0), target_weight = 32500 * 4)
```

Comment:
After https://github.com/bitcoin/bitcoin/pull/30718 this should likely be:
```suggestion
                                                utxo_to_spend=confirmed_utxos.pop(0), target_vsize=32500)
```

Reply:
This is introduced here incorrectly and fixed in a later commit - as discussed, could you please add it in the same or after the fix commit?